### PR TITLE
modified Copter-4.1.5 for JFB-100

### PR DESCRIPTION
**Pull request by jfbblue0922**
It is a program for the development board JFB-100 for JAE. The IMU is equipped with SCHA63T (MURATA).

### CODE REVIEW

File: libraries/AP_HAL_ChibiOS/hwdef/JFB100/README.md [hunk 1]
```diff
@@ -0,0 +1,393 @@
+# Pixhawk4 Flight Controller
```
Comment (amilcarlucas) [inline libraries/AP_HAL_ChibiOS/hwdef/JFB100/README.md:1]: This should be JB100 right?

File: libraries/AP_HAL_ChibiOS/hwdef/JFB100/defaults.parm [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+# Pixhawk4 does not have hw flow control on radios
```
Comment (amilcarlucas) [inline libraries/AP_HAL_ChibiOS/hwdef/JFB100/defaults.parm:2]: Pixhawk4 ????

**Review by amilcarlucas — CHANGES_REQUESTED** [review]

File: libraries/AP_HAL_ChibiOS/hwdef/JFB100/README.md [hunk 3]
```diff
@@ -0,0 +1,393 @@
...
+
+The Pixhawk4 flight controller is sold by [Holybro](http://www.holybro.com/product/pixhawk-4)
```
[the remark this answers is not in the document]
Comment (jfbblue0922) [reply libraries/AP_HAL_ChibiOS/hwdef/JFB100/README.md]: Thank you.
I will Modify and commit.

File: libraries/AP_HAL_ChibiOS/hwdef/JFB100/defaults.parm (hunk 2 above)
Comment (jfbblue0922) [reply libraries/AP_HAL_ChibiOS/hwdef/JFB100/defaults.parm]: Thank you.
I will Modify and commit.

**Comment by rmackay9** [comment]
I don't know the background of this PR but in general I agree with @amilcarlucas that it is normally best to raise PRs against master instead of a branch.

**Comment by jfbblue0922** [comment]
Sorry, I made a mistake with the pull request.
I will Close pull request and retry soon.

**Pull request closed without merging**